src/model/reducers/app-navigation-action.ts

```diff
@@ -0,0 +1,29 @@
+export const SET_ACTIVE_TAB = 'setActiveTab'
+export const SET_CONNECTION_STATUS = 'setConnectionStatus'
+export const TOGGLE_SHOW_SETTINGS = 'toggleShowSettings'
+
+export function setActiveTab(tabIndex: number): {
+    type: string
+    tabIndex: number
+} {
+    return {
+        type: SET_ACTIVE_TAB,
+        tabIndex: tabIndex,
+    }
+}
+
+export function setConnectionStatus(isConnected: boolean): {
+    type: string
+    isConnected: boolean
+} {
+    return {
+        type: SET_CONNECTION_STATUS,
+        isConnected: isConnected,
```

Comment:
You can use shorthand notation:
```suggestion
        isConnected,
```

Reply:
See commnet on https://github.com/tv2/casparcg-cliptool/pull/81#discussion_r1211562414